Vagrantfile

```diff
@@ -84,17 +188,20 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   # Kubernetes minion
   $num_minion.times do |n|
-    config.vm.define "minion-#{n+1}" do |minion|
-      customize_vm minion
+    minion_vm_name = "minion-#{n+1}"
+    minion_prefix = ENV['INSTANCE_PREFIX'] || 'kubernetes' # must mirror default in cluster/vagrant/config-default.sh
+    minion_hostname = "#{minion_prefix}#{minion_vm_name}"
```

Comment:
The minion_hostname changes with this change from `kubernetes-minion-1` to `kubernetesminion-1`.

Please address.